keel-core/src/main/kotlin/com/netflix/spinnaker/keel/core/Validations.kt

```diff
@@ -0,0 +1,10 @@
+package com.netflix.spinnaker.keel.core
```

Comment:
nit: Generally I think we name kotlin modules with a lower-case first letter.

Reply:
fixed